packages/ui-dom-utils/src/index.ts

```diff
@@ -41,7 +41,6 @@ export type { RectType } from './getBoundingClientRect'
 export { getClassList } from './getClassList'
 export { getComputedStyle } from './getComputedStyle'
 export { getFontSize } from './getFontSize'
-export { getFrameDocumentSafe } from './getFrameDocumentSafe'
```

Comment:
I deleted this because it was a trivial function, was used in 1 place, was not shown in the documentation and no one uses it internally.